ccd-definition/ComplexTypes/CaseLocationDJ.json

```diff
@@ -1,20 +1,20 @@
 [
 
   {
-    "ID": "CaseLocation",
+    "ID": "CaseLocationCivil",
     "CaseEventID": "DEFAULT_JUDGEMENT",
     "CaseFieldID": "region",
     "ListElementCode": "region",
-    "ElementLabel": " ",
+    "ElementLabel": "Reason",
```

Comment:
Reason? Should this not be Region?

Reply:
yes, it should be Region, we will fix it. Thanks